project-api/project-api-test/src/test/java/io/ballerina/projects/test/TestBuildProject.java

```diff
@@ -351,6 +353,36 @@ public void testJBallerinaBackend() {
         Assert.assertEquals(platformLibraries.size(), 4);
     }
 
+    @Test(description = "tests langlib files are not included in the jar file collection for executing the test")
```

Comment:
```suggestion
    @Test(description = "tests compiling a build project with no langlib files included in the jar file collection for executing the test")
```

Reply:
changed accordingly in 109da6bb831653d9925d85160a0853cab5a0fe77 commit